src/components/shared/Promotional/Promotional.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+export const Promotional = ({
+  title,
+  description,
+  features,
+  strongText,
+  actionText,
+  actionUrl,
+  iconUrl,
+  imageUrl,
+}) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  return (
+    <section className="dp-gray-page p-t-54 p-b-54">
+      <div className="dp-container">
+        <div className="dp-rowflex">
+          <div className="col-sm-12">
+            <div className="dp-icon-promotion">
+              <img src={_('common.ui_library_image', { imageUrl: iconUrl })} alt="icon" />
```

Comment:
Instead of icon, you can put something more descriptive and that does not use the words image, icon, etc.

Reply:
We can use logo and preview